components/resources/rewards_strings.grdp

```diff
@@ -33,6 +33,9 @@
   <message name="IDS_REWARDS_WALLET_LOG_INTO_YOUR_ACCOUNT" desc="">
     Log in to your <ph name="PROVIDER">$1<ex>Uphold</ex></ph> account
   </message>
+  <message name="IDS_REWARDS_WALLET_MANAGE_ADS" desc="">
+    Manage ads
```

Comment:
Not `Manage Brave Ads`?

Reply:
Right - this is for the tooltip text [here](https://www.figma.com/file/A77Iv5PtEygWFwI0cfV6f1/Estimated-earnings-counter?node-id=1001%3A36142&mode=dev).